piff/interp.py

```diff
@@ -173,7 +173,7 @@ def _finish_write(self, fits, extname):
         :param fits:        An open fitsio.FITS object
         :param extname:     The base name of the extension
         """
-        pass
+        raise NotImplementedError("Derived classes must define the _finish_write method.")
```

Comment:
I think that was the rationale.  But given that even `MeanInterp` implements this, I suspect all interpolant classes will need to do something here.  

And if not, it's probably a good idea to explicitly override this method and just have it `pass` with a comment saying why nothing else is required.